mlflow/models/evaluation/base.py

```diff
@@ -1102,10 +1116,28 @@ def evaluate(
         .. _rouge_score:
             https://pypi.org/project/rouge-score
 
+        .. _transformers:
+            https://pypi.org/project/transformers
+
+        .. _textstat:
+            https://pypi.org/project/textstat
```

Comment:
Looks like this project hasn't been updated in > 1 year and has less than 1k GitHub stars. Are there more popular / reputable sources where we can get the same metrics, e.g. HuggingFace's `evaluate` library, Stanford HELM, OpenAI evaluations, scikit-learn (unlikely)?

Reply:
Having trouble finding anything better...I'm okay with using this for now :)